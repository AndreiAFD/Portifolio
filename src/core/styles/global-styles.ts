import { createGlobalStyle } from "styled-components";

export const GlobalStyles = createGlobalStyle`
@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');

  * { 
    margin: 0;
    padding: 0;
    box-sizing: border-box;
    font-family: 'Inter', sans-serif;
    
  }

  html, body, #root {
    height: 100%;
    background-color: #EBEBEB;
  }

  @keyframes from-left-about {
    from {
      opacity: 0;
      margin-left: -100px;
    }

    to {
      opacity: 1;
      margin-left: 20px;
    }
  }


  @keyframes from-left-portifolio {
    from {
      opacity: 0;
      margin-left: -100px;
    }

    to {
      opacity: 1;
      margin-left: 20px;
    }
  }

  @keyframes from-left-contact {
    from {
      opacity: 0;
      margin-left: -100px;
    }

    to {
      opacity: 1;
      margin-left: 20px;
    }
  }

  @keyframes title-header-shaddow {
    from {
      bottom: -200px;
      transform: rotate(4deg);
    }

    to {
      bottom: -90px;
      transform: rotate(-2deg);

    }
  }

`;
