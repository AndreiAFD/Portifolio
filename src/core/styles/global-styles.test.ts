import React from "react";
import { GlobalStyles } from "./global-styles";

describe("GlobalStyles", () => {
  it("should render correctly", () => {
    expect(GlobalStyles).toMatchSnapshot();
  });
});
