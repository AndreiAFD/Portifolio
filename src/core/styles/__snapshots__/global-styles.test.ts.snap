// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalStyles should render correctly 1`] = `
Object {
  "$$typeof": Symbol(react.memo),
  "compare": null,
  "type": [Function],
}
`;
